probe-rs/src/probe/arm_debug_interface.rs

```diff
@@ -92,9 +92,6 @@ pub struct ProbeStatistics {
 
     /// Number of SWD FAULT responses encountered.
     num_faults: usize,
-
-    /// Number of line resets executed.
-    num_line_resets: usize,
```

Comment:
Don't we want to keep those stats and if they are broken, fix them?

Reply:
Are these even useful at all?

Reply:
We built those in to benchmark our protocol implementations a bit more to maybe find some inefficiencies :)

Reply:
Okay so how do you suggest dealing with this particular change now that line_reset is not implemented here? :)